example_configs/mailcow.md

```diff
@@ -0,0 +1,34 @@
+# Mailcow LLDAP example config
+
+## Setting location
+First, go to `System` > `Configuration`.
+
+And then, `Access` > `Identity Provider`.
+
+## Configuration
+
+Set `Identity Provider` as `LDAP`
+
+Set `Host` as the LLDAP IP.
+
+Set `Port` as the LLDAP port, which should be `3890` by default.
+
+Set `Use SSL`, `Use StartTLS`, `Ignore SSL Errors` to `False` if you did not enable SSL for LLDAP.
```

Comment:
Maybe mention that otherwise it's SSL, not StartTLS, and that if you have a properly-signed certificate (not self-signed, or with the proper root installed) you shouldn't ignore errors.